README.md

```diff
@@ -16,18 +16,18 @@ cryptographic and security reviews.
-| [Serpent](https://en.wikipedia.org/wiki/Serpent_(cipher)) | `serpent` | [![crates.io](https://img.shields.io/crates/v/serpent.svg)](https://crates.io/crates/serpent) | [![Documentation](https://docs.rs/serpent/badge.svg)](https://docs.rs/serpent) | ![build](https://github.com/RustCrypto/block-ciphers/workflows/serpent/badge.svg?branch=master&event=push)
-| [SM4](https://en.wikipedia.org/wiki/SM4_(cipher)) | `sm4` | [![crates.io](https://img.shields.io/crates/v/sm4.svg)](https://crates.io/crates/sm4) | [![Documentation](https://docs.rs/sm4/badge.svg)](https://docs.rs/sm4) | ![build](https://github.com/RustCrypto/block-ciphers/workflows/sm4/badge.svg?branch=master&event=push)
-| [Twofish](https://en.wikipedia.org/wiki/Twofish) | `twofish` | [![crates.io](https://img.shields.io/crates/v/twofish.svg)](https://crates.io/crates/twofish) | [![Documentation](https://docs.rs/twofish/badge.svg)](https://docs.rs/twofish) | ![build](https://github.com/RustCrypto/block-ciphers/workflows/twofish/badge.svg?branch=master&event=push)
-| [Threefish](https://en.wikipedia.org/wiki/Threefish) | `threefish` | [![crates.io](https://img.shields.io/crates/v/threefish.svg)](https://crates.io/crates/threefish) | [![Documentation](https://docs.rs/threefish/badge.svg)](https://docs.rs/threefish) | ![build](https://github.com/RustCrypto/block-ciphers/workflows/threefish/badge.svg?branch=master&event=push)
+| [AES](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard) (Rijndael) | `aes` <br/><br/> `aesni` <br/><br/> `aes-soft` | [![crates.io](https://img.shields.io/crates/v/aes.svg)](https://crates.io/crates/aes) <br/><br/> [![crates.io](https://img.shields.io/crates/v/aesni.svg)](https://crates.io/crates/aesni)  <br/><br/> [![crates.io](https://img.shields.io/crates/v/aes-soft.svg)](https://crates.io/crates/aes-soft) | [![Documentation](https://docs.rs/aes/badge.svg)](https://docs.rs/aes)  <br/><br/> [![Documentation](https://docs.rs/aesni/badge.svg)](https://docs.rs/aesni)  <br/><br/> [![Documentation](https://docs.rs/aes-soft/badge.svg)](https://docs.rs/aes-soft) | [![aes build](https://github.com/RustCrypto/block-ciphers/workflows/aes/badge.svg?branch=master&event=push)](https://github.com/RustCrypto/block-ciphers/actions?query=workflow:aes+branch:master) <br/><br/> [![aesni build](https://github.com/RustCrypto/block-ciphers/workflows/aesni/badge.svg?branch=master&event=push)]](https://github.com/RustCrypto/block-ciphers/actions?query=workflow:aesni+branch:master) <br/><br/> [![aes-soft build](https://github.com/RustCrypto/block-ciphers/workflows/aes-soft/badge.svg)](https://github.com/RustCrypto/block-ciphers/actions?query=workflow:aes-soft+branch:master)
```

Comment:
It looks like the `aesni` badge isn't rendering correctly (due to an extra `]` it appears)

<img width="446" alt="Screen Shot 2020-06-03 at 6 29 17 AM" src="https://user-images.githubusercontent.com/797/83642374-9cd87c00-a563-11ea-8d7e-a7b7439efbe9.png">